### ISSUE TITLE
Air vs subs - do not offer option to submerge/remain if submerge is automatic

### ISSUE THREAD

**Issue by DanVanAtta** [triplea-game/triplea#6101]
**How can the problem be recreated?**
- send fighters vs a navy containing at least one sub
- notice there is an option to submerge or remain
- subs will always automatically submerge

**When the problem is fixed, what should have happened instead?**
- do not offer choice for subs to submerge, just submerge them.

**Comment by panther2** [comment]
Why do they even have to submerge? In all rulesets where they can't hit air units and cannot be hit by air units they don't even need to submerge.

Or is this just the program logic to keep them off the battle?

**Comment by Cernelius** [comment]
Yeah, assuming that by "send fighters" you mean "**send fighters and nothing else into a hostile sea zone without units of yours already in there**", as @panther2 says, it's not like submarines submerge automatically, it is they cannot be hit by air and they cannot hit air, thus they remain there doing nothing, if not submerged, which they don't have to and it's irrelevant (if it happens that only air remain in offence and only submarines remain in defence, then the battle ends immediately, for both sides having no possible targets, **practically just like the 0 vs 0 power case**, in this case, if the submarines never decided to submerge).
